<launch>	
	<node name="image_drone" pkg="kocheng" output="screen" type="image_drone" />
	<node name="flag_drone" pkg="kocheng" output="screen" type="flag_drone" />
</launch>
